pyiode/iode_database/abstract_database.pyx

```diff
@@ -501,6 +508,17 @@ cdef class _AbstractDatabase:
         input_file = str(Path(input_file).resolve())
         self.abstract_db_ptr.merge_into(input_file.encode())
 
+    # GDM>
+    # * I think the API would be nicer if that information was available as properties on the "single object" classes
+    #   (e.g. Equation, Scalar, Table). You might need to create classes for Comment, Identity, List and Variable,
+    #   but this seems like a good thing to me anyway.
+    #   For example,
+    #     >>> variables.get_associated_objects_list("AOUC", COMMENTS)
+    #   would become:
+    #     >>> variables["AOUC"].comments
+    # * the warning in the doctest seems very useful, given that I fail to see the difference but it lacks enough
+    #   explanation (for me) to understand those differences. In short, the
```

Comment:
The last sentence is gibberish. It should be:
In short, since I don't understand everything about this, maybe the comment about making it properties on the "single object" classes is not pertinent.

Reply:
It is a partial implementation of the fonction `DataSearch`. Actually, I have to rewrite the C++ method `get_associated_objects_list` as `search` or `search_in` and add the arguments of the `DataSearch` report fonction: 
```
DataSearch mask word ecase in_name in_formula in_text list_result type
```

Reply:
See PR https://github.com/plan-be/iode/pull/553